taxonomy-manager-rest/src/main/java/com/digirati/taxman/rest/taxonomy/ConceptResource.java

```diff
@@ -29,6 +31,12 @@
     @JsonLdFrame(input = "/jsonld/framing/concept.json")
     Response getConcept(@BeanParam ConceptPath params);
 
+    @GET
+    @Path("/by-label/{label}")
```

Comment:
This should be something like `/search`, with a query parameter deciding what the search term is. E.g., ```search?label=whatever```. Then we can extend it in the future without making breaking changes.

Reply:
cc @stephenwf

Reply:
> Everyone happy with that or do we want to scope it to a single concept scheme as per the mock api? (edited) 

Probably not scoped to a concept scheme. If you scope it to a concept scheme, there's no way for the UI to let you create cross-concept scheme references.